feature-libs/user/profile/components/update-password/update-password-component.service.ts

```diff
@@ -78,7 +75,7 @@ export class UpdatePasswordComponentService {
 
   protected onSuccess(): void {
     this.globalMessageService.add(
-      { key: 'updatePasswordForm.passwordUpdateSuccess' },
+      { key: 'myAccountV2PasswordForm.passwordUpdateSuccess' },
```

Comment:
[MEDIUM]
Because this componet-service is shared in between component v1 and v2, the i18n key used should be conditional, based on the feature flag.

PS. if we're going to inject a feature flag to the class, please:
- let's use `inject()` function 
- let's make the property of the feature flag `private`, not to pollute the Public API surface, and to be able to seamlessly remove this property in the future without a breaking change

Note: similar remark applies to other places in component-services in this PR

Reply:
Fixed.